lobster/tools/codebeamer/codebeamer.py

```diff
@@ -522,17 +534,25 @@ def main():
 
     elif cb_config.get("import_query"):
         try:
-            query_id = int(cb_config["import_query"])
-        except ValueError:
-            sys.exit("lobster-codebeamer: Query ID must be an integer.")
-        if query_id < 1:
-            sys.exit("lobster-codebeamer: Query ID must be a positive integer.")
+            if (isinstance(cb_config["import_query"], int) and
+                cb_config["import_query"] < 0):
+                ap.error("import-query must be a positive integer")
+            elif isinstance(cb_config["import_query"], str):
+                if (cb_config["import_query"].startswith("-") and
+                    cb_config["import_query"][1:].isdigit()):
+                    ap.error("import-query must be a positive integer")
+                elif cb_config["import_query"].startswith("-"):
+                    ap.error("import-query must be a valid cbQL query")
```

Comment:
```suggestion
                if cb_config["import_query"].startswith("-"):
                    if cb_config["import_query"][1:].isdigit()):
                       ap.error("import-query must be a positive integer")
                    else:
                       ap.error("import-query must be a valid cbQL query")
```

You can simplify this one.

Reply:
Here, this code also does not work if the user input is as below,
`import_query: "271679129"`

This input is then considered as a "query string"

we must convert the ID provided as string to int